lib/models/addons-factory.js

```diff
@@ -59,7 +59,7 @@ class AddonsFactory {
         } catch (e) {
           project.ui.writeError(e);
           const SilentError = require('silent-error');
-          throw new SilentError(`An error occured in the constructor for ${addonInfo.name} at ${addonInfo.path}`);
+          throw new SilentError(`An error occurred in the constructor for ${addonInfo.name} at ${addonInfo.path}`);
```

Comment:
ya this is a non-issue.

If we have this issue, we should likely ensure every error has a specific and stable code or name. If that is something that interests someone, lets open an RFC and go from there.